restaurant-billing-app/backend/.env

```
@@ -1,6 +1,6 @@
 DB_USER=postgres
 DB_PASSWORD=Deepthi@2004
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)